### PR TITLE
Savedata

### PR DESCRIPTION
**Pull request by MariaRiosNavarro**
Please Check:

- wrap the new `addSVG` with a Link at the ContactList to make possible the path to the new /create page (with `CreateContactForm`)
- Create the `CreateContactForm` to add new Contacts. Only the name is required.
- Create Schema/Model & connect. Create  .env.local file and check in .gitignore .
- Create API route.
- SWR to fetch in _app

**Note**: the Reading of the saved data is the next US, that's why in the Contact List we only see the DB data in the project. But the success message is attached to the logic of the data coming to MongoAtlas. This way you can verify that it works

[Deployment](https://mobile-simulation-prop-1ch7m0c02-mariariosnavarro.vercel.app/create)

### CODE REVIEW

File: pages/api/contacts/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      const contactData = request.body;
+      await Contact.create(contactData);
+      response.status(201).json({ message: "Contact created" });
+    } catch (error) {
+      console.log(error);
+      response.status(400).json({ error: error.messge });
+    }
+  }
+
+  if (request.method === "GET") {
```
Comment (actyvystom) [inline pages/api/contacts/index.js:18]: To conform to a correct error handling you should also add a `try / catch` statement to your get request block.

**Review by actyvystom — CHANGES_REQUESTED** [review]
Nice work! Keep on hacking! 🚀 
See just a tiny comment concerning error handling in your api route. 
Cheers, Thomas

File: pages/api/contacts/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    const contacts = await Contact.find();
+    response.status(200).json(contacts);
+    return;
```
Comment (actyvystom) [inline pages/api/contacts/index.js]: To conform to a correct error handling you should also add a `try / catch` statement to your get request block.

Comment (MariaRiosNavarro) [reply pages/api/contacts/index.js]: Done 👍

**Review by sebastianwel — APPROVED** [review]

**Comment by sebastianwel** [comment]
Looks good to me! Nice Code.

**Pull request merged**